code/game/asteroid.dm

```diff
@@ -1,6 +1,6 @@
 var/global/list/possiblethemes = list("organharvest","cult","wizden","cavein","xenoden","hitech","speakeasy","plantlab", "standroom")
 
-var/global/max_secret_rooms = 10
+var/global/max_secret_rooms = 12
```

Comment:
I guess it's just for "if someone else wants to make a room, this is here so it's still 100% spawn."